airbyte-integrations/connectors/source-kafka/src/test-integration/java/io/airbyte/integrations/source/kafka/KafkaSourceAcceptanceTest.java

```diff
@@ -96,14 +108,18 @@ private void sendEvent() throws ExecutionException, InterruptedException {
 
   private void createTopic() throws Exception {
     try (final var admin = AdminClient.create(Map.of(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, KAFKA.getBootstrapServers()))) {
-      final NewTopic topic = new NewTopic(TOPIC_NAME, 1, (short) 1);
+      final NewTopic topic = new NewTopic(topicName, 1, (short) 1);
       admin.createTopics(Collections.singletonList(topic)).all().get();
     }
   }
 
   @Override
   protected void tearDown(final TestDestinationEnv testEnv) {
-    KAFKA.close();
+    try (final var admin = AdminClient.create(Map.of(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, KAFKA.getBootstrapServers()))) {
+      admin.deleteTopics(List.of(topicName)).all().get();
+    } catch (Exception e) {
+      throw new RuntimeException(e);
+    }
```

Comment:
What's this do, just re-type the exception?

Reply:
I think tearDown is not supposed to throw any exceptions. But sometimes, the tearDown failing means the whole suit will be a problem, so we're throwing a RuntimeException. Not the cleanest thing, but we don't have a clean story around exception handling, either in tests or in production code